merlin/models/tf/core/tabular.py

```diff
@@ -597,6 +606,17 @@ def get_config(self):
 
         return config
 
+    def select_by_tag(self, tags: Tags) -> Optional["Filter"]:
+        if isinstance(self.feature_names, Tags):
+            schema = self.schema.select_by_tag(self.feature_names).select_by_tag(tags)
+        elif isinstance(self.feature_names, collections.Sequence):
+            schema = self.schema.select_by_name(self.feature_names).select_by_tag(tags)
+        else:
+            schema = self.schema.select_by_tag(tags)
```

Comment:
in what scenario would this else be reached?

Reply:
Never in theory :) Replaced it with raising an error.